server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java

```diff
@@ -69,13 +69,22 @@ public class Upgrader11to12 implements Upgrader {
   static final Set<Text> UPGRADE_FAMILIES =
       Set.of(DataFileColumnFamily.NAME, CHOPPED, ExternalCompactionColumnFamily.NAME);
 
+  public static final String ZTRACERS = "/tracers";
+  public static final String ZCONF_LEGACY = "/conf";
+
   @Override
   public void upgradeZookeeper(@NonNull ServerContext context) {
     log.debug("Upgrade ZooKeeper: upgrading to data version {}", METADATA_FILE_JSON_ENCODING);
-    var rootBase = ZooUtil.getRoot(context.getInstanceID()) + ZROOT_TABLET;
+    var zooRoot = ZooUtil.getRoot(context.getInstanceID());
+    var rootBase = zooRoot + ZROOT_TABLET;
 
     try {
       var zrw = context.getZooReaderWriter();
+
+      // clean up nodes no longer in use
+      zrw.recursiveDelete(zooRoot + ZTRACERS, ZooUtil.NodeMissingPolicy.SKIP);
+      zrw.recursiveDelete(zooRoot + ZCONF_LEGACY, ZooUtil.NodeMissingPolicy.SKIP);
```

Comment:
@ctubbsii @EdColeman  is it ok to remove this zookeeper node in the 3.1 upgrade?

Reply:
I looked into this a bit and do not think there is anything to actually remove in zookeeper.  Looking at [ConfigPropertyUpgrader](https://github.com/apache/accumulo/blob/0e7417d0369aca85e149563cc52ad7c9108dfe6d/server/base/src/main/java/org/apache/accumulo/server/conf/util/ConfigPropertyUpgrader.java) in 2.1 ZCONF_LEGACY is only used for table and namespace config AND the upgrade sets a boolean to remove those.

Reply:
Going to update this PR to only remove the constant.